lib/Root.js

```diff
@@ -2,7 +2,8 @@ import React, { PropTypes } from 'react'
 import { Redirect, Router, Route } from 'react-router'
 import { Provider } from 'redux/react'
 import { createDispatcher, createRedux, composeStores } from 'redux'
-import { loggerMiddleware, thunkMiddleware } from './middleware'
+import thunkMiddleware from 'redux/lib/middleware/thunk'
```

Comment:
I don't have a strong opinion here. We're going to slightly tweak API before 1.0 anyway.